curriculum/structure/blocks/lab-debug-type-coercion-errors.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "Debug Type Coercion Errors in a Buggy App",
+  "isUpcomingChange": true,
+  "dashedName": "lab-debug-type-coercion-errors",
+  "helpCategory": "JavaScript",
+  "blockLayout": "link",
+  "blockType": "lab",
```

```suggestion
  "blockLabel": "lab",
```

per https://github.com/freeCodeCamp/freeCodeCamp/pull/62548